src/benchq/resource_estimation/_footprint_analysis.py

```diff
@@ -22,7 +22,7 @@
 class MagicStateFactory:
     details: str
     physical_qubit_footprint: int
-    rounds: int
+    distillation_time_in_cycles: int
```

Comment:
It's a bit weird for a time be in integers. Plus  in azure `cycle_time` is a float and `surface_code_cycle_time` is also a float.